react/components/ShippingInfo.tsx

```diff
@@ -7,22 +7,18 @@ interface CustomProps {
 }
 
 const ShippingInfo: FunctionComponent<CustomProps> = ({ option }) => {
+  const optionId = option.id.toLowerCase().replace(/\s+/g, '')
```

Comment:
Maybe it's better to use a package like [slugify](https://www.npmjs.com/package/slugify) to do this job. At least move this `.toLowerCase().replace(/\s+/g, '')` logic to its own function, otherwise if in the future you need to change this string transformation logic you would have to do so in multiple places.